### DISCUSSION TITLE
Can you add the this feature

### DISCUSSION THREAD

**Discussion by 0killdude** [danielkrupinski/Osiris#3186] [Ideas]
stattrak, wear, souvenir and stickers directly to your weapon like other inventory changer, i have to tell you if this done this could compete with some of the best paid cheats of csgo and im not exaggerating

**Comment by danielkrupinski** [comment]
You can apply stickers in CS:GO inventory.

**Reply by PEXCH-SENTXMENT** [reply]
No, he meant that the quality and float could be set in the cheat when adding or already added items
![image](https://user-images.githubusercontent.com/63977167/128852998-4fe9247e-3127-4ff7-9ec7-c607accfbb00.png)

**Reply by 0killdude** [reply]
what cheat are you using

**Reply by PEXCH-SENTXMENT** [reply]
Osiris

**Reply by PEXCH-SENTXMENT** [reply]
https://www.dropbox.com/s/437lez9ibfg3ob0/cheat.dll?dl=0
But configs from official osiris will not work
Here are the configs
https://www.dropbox.com/s/yng0bkke8c5292e/Peach.XYZ.zip?dl=0

**Comment by 0killdude** [comment]
yes thats what i meant sorry for using bad grammar, but this feature could really make your cheat on of the best

**Reply by linuxer9** [reply]
you could change that in config file.

**Reply by 0killdude** [reply]
yea i know i was hoping that it could be done while adding the weapons itself

**Comment by PEXCH-SENTXMENT** [comment]
This is a crappy idea, but I don't think it will be done because I don't know why XDXDXD
